tools/log_simple.c

```diff
@@ -0,0 +1,82 @@
+/*
+ * simple logger to stdout/stderr
```

Comment:
Could you rename this tool to `ubx_log`? It's simple right now, but it will likely grow in the future...

Reply:
Done